docs/src/prototypes/chatMessages/ControlMessages/ControlMessage.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react'
+import { Ref, ChatMessageProps, Chat, Icon } from '@stardust-ui/react'
+
+interface ControlMessageProps {
+  focused?: boolean
+  icon?: boolean
+  message: ChatMessageProps
+}
+class ControlMessage extends React.Component<ControlMessageProps> {
```

Comment:
I'm wondering if it would make sense to use createComponent here. It might be a nice example.

Reply:
discussed w/ @mnajdova , doesn't make much sense here